presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -121,6 +121,8 @@
     public static final String ENABLE_DYNAMIC_FILTERING = "enable_dynamic_filtering";
     public static final String QUERY_MAX_MEMORY_PER_NODE = "query_max_memory_per_node";
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
+    public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_VALUE_COUNT = "dynamic_filtering_max_per_driver_value_count";
+    public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_BYTE_SIZE = "dynamic_filtering_max_per_driver_byte_size";
```

Comment:
Since the unit for this is DataSize now, `DYNAMIC_FILTERING_MAX_PER_DRIVER_SIZE = "dynamic_filtering_max_per_driver_size"` would be a more suitable name

Reply:
Thanks, fixed.

Reply:
Renamed a few more instances of `ByteSize` into `Size`.